### PR TITLE
Fix build badges in README

### PR DESCRIPTION
**Pull request by RobertWilbrandt**
The previous ones didn't work any more, so i tried finding the correct badges. It would be nice if someone checked the jenkins urls, they seemed correct to me but i don't really know anything about jenkins.ros.org.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -16,9 +16,9 @@ Developed by [Dave Coleman](http://dav.ee/) at the University of Colorado Boulde
...
 
- * [![Build Status](https://travis-ci.org/davetcoleman/ros_control_boilerplate.svg)](https://travis-ci.org/davetcoleman/ros_control_boilerplate) Travis CI
- * [![Devel Job Status](http://jenkins.ros.org/buildStatus/icon?job=devel-indigo-ros_control_boilerplate)](http://jenkins.ros.org/job/devel-indigo-ros_control_boilerplate) Devel Job Status
- * [![Build Status](http://jenkins.ros.org/buildStatus/icon?job=ros-indigo-ros-control-boilerplate_binarydeb_trusty_amd64)](http://jenkins.ros.org/job/ros-indigo-ros-control-boilerplate_binarydeb_trusty_amd64/) AMD64 Debian Job Status
+ * [![Build Status](https://travis-ci.org/PickNikRobotics/ros_control_boilerplate.svg?branch=melodic-devel)](https://travis-ci.org/PickNikRobotics/ros_control_boilerplate) Travis CI
+ * [![Build Status](http://build.ros.org/buildStatus/icon?subject=Devel%20Job%20Status&job=Mdev__ros_control_boilerplate__ubuntu_bionic_amd64)](http://build.ros.org/job/Mdev__ros_control_boilerplate__ubuntu_bionic_amd64/)
```
Comment (JafarAbdi) [inline README.md]:
```suggestion
 * [![Devel Job Status](http://build.ros.org/buildStatus/icon?subject=Devel%20Job%20Status&job=Mdev__ros_control_boilerplate__ubuntu_bionic_amd64)](http://build.ros.org/job/Mdev__ros_control_boilerplate__ubuntu_bionic_amd64/) Devel Job Status
```

File: README.md [hunk 2]
```diff
@@ -16,9 +16,9 @@ Developed by [Dave Coleman](http://dav.ee/) at the University of Colorado Boulde
...
+ * [![Build Status](http://build.ros.org/buildStatus/icon?subject=AMD64%20Job%20Status&job=Mbin_uB64__ros_control_boilerplate__ubuntu_bionic_amd64__binary)](http://build.ros.org/job/Mbin_uB64__ros_control_boilerplate__ubuntu_bionic_amd64__binary/)
```
Comment (JafarAbdi) [inline README.md]:
```suggestion
 * [![Build Status](http://build.ros.org/buildStatus/icon?subject=AMD64%20Job%20Status&job=Mbin_uB64__ros_control_boilerplate__ubuntu_bionic_amd64__binary)](http://build.ros.org/job/Mbin_uB64__ros_control_boilerplate__ubuntu_bionic_amd64__binary/) AMD64 Debian Job Status
```

**Review by JafarAbdi — CHANGES_REQUESTED** [review]
LGTM, could you please restore some of the changes as suggested .? Thanks!

File: README.md (hunk 1 above)
Comment (RobertWilbrandt) [reply README.md]: can do, in that case i'll remove the badge label again as well

**Review by JafarAbdi — APPROVED** [review]

**Pull request merged**